src/server/svr_jobfunc.c

```diff
@@ -3395,6 +3395,10 @@ resv_setResvState(resc_resv *presv, int state, int sub)
 	    (presv->ri_qs.ri_substate == sub))
 		return;
 
+	DBPRT(("resv_name=%s, o_state=%d, o_sub=%d, state=%d, sub=%d",
```

Comment:
This might be useful as a log message at a higher log level rather than a debug print.